doc/Caliptra.md

```diff
@@ -866,65 +866,76 @@ Caliptra firmware is composed of two images: an FMC image and an application fir
 *Figure 7: Firmware image layout*
 
 To verify the firmware, Caliptra ROM performs the following steps:
```

Comment:
nit: Since we now have multiple image formats I think we need to first check which image to use. Not sure if we want to include that here or just keep it as a ROM implementation detail.

Reply:
We can keep this detail in the ROM.